### PR TITLE
Change non-existent World to World3D

### PR DESCRIPTION
**Pull request by qarmin**
Prevent from errors when clicking at Viewport node
```
Cannot get class 'World'.
```
also changed occurrences of ..world() to ...world_3d()

### CODE REVIEW

**Comment by YeldhamDev** [comment]
> Maybe set_world should be changed to set_world3d?

Considering the recent changes and the existence of a `set_world_2d()`, absolutely.

**Comment by akien-mga** [comment]
There's also `own_world` and basically all identifiers referring to `wormd` in that class.

File: scene/main/viewport.cpp [hunk 1]
```diff
@@ -3438,7 +3438,7 @@ void Viewport::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("get_default_canvas_item_texture_repeat"), &Viewport::get_default_canvas_item_texture_repeat);
 
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "own_world"), "set_use_own_world", "is_using_own_world");
-	ADD_PROPERTY(PropertyInfo(Variant::OBJECT, "world", PROPERTY_HINT_RESOURCE_TYPE, "World"), "set_world", "get_world");
+	ADD_PROPERTY(PropertyInfo(Variant::OBJECT, "world", PROPERTY_HINT_RESOURCE_TYPE, "World3D"), "set_world", "get_world");
```
Comment (aaronfranke) [inline scene/main/viewport.cpp:3441]: `"world"` should also be `"world_3d"` just like `"world_2d"` below.

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**